src/features/snap/components/community-sentiment/CommunitySentimentModal.tsx

```diff
@@ -0,0 +1,139 @@
+        suffixText = t`has been evaluated as unsecured by your community`;
+        break;
+      default:
+        suffixText = t`is currently under review by your community and may be insecure`;
+        break;
+    }
+    return (
+      <Trans>
+        <Text variant="blue" as="span">
+          {snapName}
+        </Text>{' '}
+        {suffixText}
+      </Trans>
+    );
+  }, [sentiment, snapName]);
+
+  const getHeaderIcon = useCallback(() => {
+    switch (sentiment.type) {
+      case SentimentType.InsufficientReview:
+        return <WarningFilledShadowIcon />;
+      case SentimentType.Secured:
+        return <CheckFilledShadowIcon />;
+      case SentimentType.InReview:
+        return <WarningFilledShadowIcon />;
+      case SentimentType.Unsecured:
+        return <WarningFilledShadowIcon fill={'error.default'} />;
+      default:
+        return <CheckFilledShadowIcon />;
+    }
+  }, [sentiment]);
+  return (
+    <SimpleModal isOpen={isOpen} onClose={onClose} headerIcon={getHeaderIcon()}>
+      <VStack textAlign="center">
+        <Text fontSize="md" fontWeight="bold">
+          {getHeaderTitle()}
+        </Text>
+        <Box
+          background="background.default"
+          padding="1rem"
+          margin={'1.5'}
+          borderRadius="1rem"
+          width={'100%'}
+        >
+          <VStack alignItems="flex-start" spacing={'1.5rem'}>
+            <HStack>
+              <VStack alignItems="flex-start">
+                <HStack>
+                  <Tag
+                    textTransform="none"
+                    height={'1.5rem'}
+                    variant={getColorForSentiment(sentiment.type)}
+                  >
+                    <SignHexagonIcon
+                      margin={-1}
+                      fill="currentColor"
+                    ></SignHexagonIcon>
+                    <TagLabel>
+                      <Trans>{`${sentiment.type} by Community`}</Trans>
+                    </TagLabel>
+                  </Tag>
+                </HStack>
+                <HStack fontSize={'sm'}>
+                  <Text variant="blue">
+                    <Trans>{sentiment.endorsements} endorsements</Trans>
+                  </Text>
+                  <Text> and </Text>
+                  <Text variant="blue">
+                    <Trans>{sentiment.reports} reports</Trans>
+                  </Text>
+                </HStack>
+              </VStack>
+            </HStack>
+            <HStack>
+              <VStack alignItems="flex-start">
+                <HStack noOfLines={2}>
+                  <Text>
+                    <Trans>SOURCE CODE</Trans>
+                  </Text>
+                </HStack>
+                <HStack>
+                  <Button
+                    variant="link"
+                    fontWeight={'light'}
+                    rightIcon={<ExternalLinkIcon />}
+                  >
+                    <Text variant="blue">
+                      <Trans>GitHub</Trans>
```

Comment:
Why add a button for Github that doesn't link to source code

Reply:
Fair point. In next sprint we are expecting the link. So I added it as button just to avoid rework.